### PR TITLE
[BUGS-6800, BUGS-7404] Release Notes: WP-CLI wrapper changes

### PR DESCRIPTION
**Pull request by jazzsequence**
## Summary

**[Release Notes](https://docs.pantheon.io/release-notes/)** - Documents how we changed our WP-CLI wrapper from its previous state and what problems that change may solve.

### Dependencies and Timing

**Release**:
- [x] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/old-path/` => `/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Review by scottbuscemi — APPROVED** [review]

File: source/releasenotes/2024-02-27-wp-cli-changes.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+title: WP-CLI Pantheon wrapper changes
+published_date: "2024-02-27"
+categories: [wordpress,action-required]
+---
+
+We've recently modified the Pantheon wrapper around the WP-CLI executable to solve a number of interrelated issues that customers may have previously been experiencing and/or using workarounds to solve.
+
+<h3>The Problem</h3>
+
+* Our WP-CLI wrapper included a hard-coded `--url` flag for all `wp` commands run on the platform (via Terminus, e.g. `terminus remote:wp` or `terminus wp` or internal workflows that relied on WP-CLI like flushing cache or viewing the Status page in the dashboard).
```
Comment (pwtyler) [inline source/releasenotes/2024-02-27-wp-cli-changes.md:11]:
```suggestion
* Our WP-CLI wrapper included a hard-coded `--url` flag for all `wp` commands run on the platform (via Terminus, e.g. `terminus remote:wp` or `terminus wp`, or internal workflows that relied on WP-CLI like flushing cache or viewing the Status page in the dashboard).
```

File: source/releasenotes/2024-02-27-wp-cli-changes.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+* For WordPress multisites, the hard-coded `--url` flag pointed to the Pantheon platform domain (`*.pantheonsite.io`) instead of the actual domain of the site resulting in errors and the command failing unless executed manually (via Terminus with the correct `--url` flag attached).
```
Comment (pwtyler) [inline source/releasenotes/2024-02-27-wp-cli-changes.md:12]: Worth clarifying this is/was primary domain set in the dashboard, or not worth the distinction?

**Comment by IngridKwok** [comment]
@jazzsequence @scottbuscemi @pwtyler Made some minor changes and added the recommended action to the intro paragraph. Please review.

**Comment by IngridKwok** [comment]
@stevector Since Chris is OOO, this can be published when ready. Thanks.

**Pull request merged**